packages/filebrowser/src/listing.ts

```diff
@@ -1221,10 +1216,15 @@ export class DirListing extends Widget {
     // when it's not needed. E.g. just moving files around
     // in a filebrowser
     if (selectedItems.length > 0) {
-      let attachmentThunks = paths.map(path => {
-        return () => services.contents.get(path);
-      });
-      this._drag.mimeData.setData(ATTACHMENTS_MIME, attachmentThunks);
+      for (let i = 0; i < selectedItems.length; i++) {
```

Comment:
Generally, we prefer the `for (let item of selectedItems) {` pattern.

Reply:
Fixed